util/tls/tls_engine_test.cc

```diff
@@ -333,11 +333,6 @@ TEST_F(SslStreamTest, Write) {
   }
 }
 
-
-TEST_F(SslStreamTest, Socket) {
```

Comment:
why did you delete it?

Reply:
because it defaults construct a `TlsSocket`, this test does nothing :D Also, TlsSocket, is no longer default constructable